webapp/src/config/configurations.ts

```diff
@@ -54,13 +54,13 @@
          logout: '',
 
          CloudformationTemplateUrl: ''
-       }
+      }
      },
      pacmanIssue: {
        CREATE_JIRA_TICKET_FOR_PACMAN_ISSUE: false, // Expected values: true || false || ''
        emailPacManIssue: {
          ISSUE_MAIL_TEMPLATE_URL: '',
-         ISSUE_EMAIL_FROM_ID: 'autofix@paladincloud.io',
+         ISSUE_EMAIL_FROM_ID: 'notify@paladincloud.io',
```

Comment:
Why do you have the email address hardcoded on the UI? API should take care of figuring it out or hard coding in API side. These type of updates must be API driven!

If this requires an API change, we should engage the api team for support.

As stated before, keep such business logic out of UI and if you encounter future scenarios of such behavior please add a task to get that dependency removed from the UI.